aspnetcore/mvc/views/tag-helpers/built-in/CacheTagHelper.md

```diff
@@ -1,6 +1,6 @@
 ﻿---
-title: Anchor Tag Helper | Microsoft Docs
-author: Peter Kellner
+title: Cache Tag Helper | Microsoft Docs
```

Comment:
Change to 
title: Cache Tag Helper in ASP.NET Core MVC
remove |  Microsoft Docs